go/libraries/doltcore/table/editor/keyless_table_editor.go

```diff
@@ -493,10 +494,10 @@ func applyEdits(ctx context.Context, tbl *doltdb.Table, acc keylessEditAcc, inde
 			return nil, err
 		}
 
-		func(k, v types.Tuple) (*doltdb.Table, error) {
+		err = func(k, v types.Tuple) (localErr error) {
```

Comment:
Fyi: https://stackoverflow.com/questions/37248898/how-does-defer-and-named-return-value-work